xbmc/FileItem.cpp

```diff
@@ -3043,7 +3043,7 @@ bool CFileItem::SkipLocalArt() const
        || StringUtils::StartsWithNoCase(m_strPath, "newsmartplaylist://")
        || StringUtils::StartsWithNoCase(m_strPath, "newplaylist://")
        || m_bIsShareOrDrive
-       || IsInternetStream()
+       || (IsInternetStream() && !URIUtils::IsOnLAN(GetPath()))
```

Comment:
better use GetDynPath()

Reply:
Have made change. Not sure why jenkins is failing. Builds and tests ok locally.